### PR TITLE
Man page fixes

### LINKED ISSUE #1401: man page spelling error
intances should be instances.

It would be nice if this wasn't gz compressed in the source, otherwise I'd have done a pull request.

### PR DESCRIPTION
**Pull request by guedou**
This PR fixes #1401.

### CODE REVIEW

**Comment by p-l-** [comment]
Don't you want to remove the gzipped file?

**Comment by p-l-** [comment]
You have `setup.py` (the tests fail).

File: doc/scapy.1 [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+arping, tcpdump, tshark, p0f, ...
+.PP
+\fBScapy\fP uses the Python interpreter as a command board. That means that 
+you can use directly Python language (assign variables, use loops, 
+define functions, etc.) If you give a file as parameter when you run 
+\fBScapy\fP, your session (variables, functions, instances, ...) will be saved 
+when you leave the interpretor, and restored the next time you launch
+\fBScapy\fP. 
+.PP
+The idea is simple. Those kind of tools do two things : sending packets
```
Comment (gpotter2) [inline doc/scapy.1]: Those kinds

File: doc/scapy.1 [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+tool.
+.PP
+\fBScapy\fP is a powerful interactive packet manipulation tool, 
+packet generator, network scanner, network discovery, packet sniffer,
+etc. It can for the moment replace hping, parts of nmap, arpspoof, arp-sk,
+arping, tcpdump, tshark, p0f, ...
+.PP
+\fBScapy\fP uses the Python interpreter as a command board. That means that 
+you can use directly Python language (assign variables, use loops, 
+define functions, etc.) If you give a file as parameter when you run
```
Comment (gpotter2) [inline doc/scapy.1]: a parameter

File: doc/scapy.1 [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+\fBScapy\fP is a powerful interactive packet manipulation tool, 
+packet generator, network scanner, network discovery, packet sniffer,
+etc. It can for the moment replace hping, parts of nmap, arpspoof, arp-sk,
+arping, tcpdump, tshark, p0f, ...
+.PP
+\fBScapy\fP uses the Python interpreter as a command board. That means that 
+you can use directly Python language (assign variables, use loops, 
+define functions, etc.) If you give a file as parameter when you run 
+\fBScapy\fP, your session (variables, functions, instances, ...) will be saved 
+when you leave the interpretor, and restored the next time you launch
```
Comment (gpotter2) [inline doc/scapy.1]: interpreter

File: doc/scapy.1 [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+.PP
+The idea is simple. Those kind of tools do two things : sending packets 
+and receiving answers. That's what \fBScapy\fP does : you define a set of 
+packets, it sends them, receives answers, matches requests with answers 
+and returns a list of packet couples (request, answer) and a list of 
+unmatched packets. This has the big advantage over tools like nmap or 
+hping that an answer is not reduced to (open/closed/filtered), but is 
+the whole packet. 
+.PP
+On top of this can be build more high level functions, for example one
```
Comment (gpotter2) [inline doc/scapy.1]: be built more high-level functions

File: doc/scapy.1 [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+.fi
+
+.SH BUGS
+Does not give the right source IP for routes that use interface aliases.
+
+May miss packets under heavy load.
+
+Session saving is limited by Python ability to marshal objects. As a 
+consequence, lambda functions and generators can't be saved, which seriously
+reduce usefulness of this feature.
```
Comment (gpotter2) [inline doc/scapy.1]: the usefulness

**Comment by gpotter2** [comment]
(grammar issues thanks to grammarly.com)

**Comment by gpotter2** [comment]
@p-l- About the failure: https://www.traviscistatus.com/incidents/0ch1w22m3lwc

**Comment by p-l-** [comment]
@gpotter2 unrelated (or at least, not totally): the build I have looked at failed because of `scapy.1.gz` file.

**Comment by p-l-** [comment]
@gpotter2 nice review!

**Pull request merged**